src/HttpBodyStream.php

```diff
@@ -64,6 +64,11 @@ public function close()
 
         $this->closed = true;
 
+        $this->input->removeListener('data', array($this, 'handleData'));
+        $this->input->removeListener('end', array($this, 'handleEnd'));
+        $this->input->removeListener('error', array($this, 'handleError'));
+        $this->input->removeListener('close', array($this, 'close'));
```

Comment:
The input will stop I just thought it would be better to remove the listeners. But ok I'll remove them.